ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/PaywallAction.kt

```diff
@@ -6,6 +6,7 @@ import dev.drewhamilton.poko.Poko
 internal sealed interface PaywallAction {
     object RestorePurchases : PaywallAction
     object NavigateBack : PaywallAction
+    object PurchasePackage : PaywallAction
```

Comment:
We discussed this over Slack but we're not passing the actual package here since the view model should already know what's being purchased.